src/vma/dev/cq_mgr.cpp

```diff
@@ -202,26 +204,15 @@ uint32_t cq_mgr::clean_cq()
 	}
 
 	return ret_total;
+#endif
```

Comment:
is it possible to do `clean_cq` for general and vmapoll configurations in the same way?